.github/workflows/ios-publish.yml

```diff
@@ -0,0 +1,154 @@
+        required: true
+      APPLE_KEY_CONTENT:
+        required: true
+      MATCH_PASSWORD:
+        required: true
+      SLACK_URL:
+        required: true
+
+jobs:
+  build_ios:
+    name: Deploying to Testflight
+    runs-on: macos-13
+
+    steps:
+      - uses: actions/checkout@v3.1.0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ inputs.NODE_VERSION }}
+
+      - name: Install npm dependencies
+        run: |
+          npm install
+
+      - name: Install Pod
+        run: |
+          cd ios
+          pod install
+          pod install
+
+      - name: Build iOS for automation on simulator
+        run: |
+          cd ios
+          fastlane build_app
+
+      - name: Upload Inji.app artifact
+        uses: actions/upload-artifact@v3.1.1
+        with:
+          name: Inji
+          path: ios/fastlane/Inji_app_file/Products/Applications/
+          retention-days: 10
+
+      - name: Deploy iOS Beta to TestFlight
+        run: |
+          cd ios
+          fastlane beta
+        env:
+          APP_STORE_CONNECT_TEAM_ID: "${{ secrets.APP_STORE_CONNECT_TEAM_ID }}"
+          DEVELOPER_APP_ID: "${{ secrets.DEVELOPER_APP_ID }}"
+          INJI_IOS_DEVELOPER_APP_IDENTIFIER: "${{ secrets.INJI_IOS_DEVELOPER_APP_IDENTIFIER }}"
+          INJI_IOS_DEVELOPER_PORTAL_TEAM_ID: "${{ secrets.INJI_IOS_DEVELOPER_PORTAL_TEAM_ID }}"
+          INJI_IOS_FASTLANE_APPLE_ID: "${{ secrets.INJI_IOS_FASTLANE_APPLE_ID }}"
+          INJI_IOS_FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD: "${{ secrets.INJI_IOS_FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD }}"
+          GIT_AUTHORIZATION: "${{ secrets.GIT_AUTHORIZATION }}"
+          INJI_IOS_PROVISIONING_PROFILE_SPECIFIER: "${{ secrets.INJI_IOS_PROVISIONING_PROFILE_SPECIFIER }}"
+          INJI_IOS_TEMP_KEYCHAIN_PASSWORD: "${{ secrets.INJI_IOS_TEMP_KEYCHAIN_PASSWORD }}"
+          INJI_IOS_TEMP_KEYCHAIN_USER: "${{ secrets.INJI_IOS_TEMP_KEYCHAIN_USER }}"
+          APPLE_KEY_ID: "${{ secrets.APPLE_KEY_ID }}"
+          APPLE_ISSUER_ID: "${{ secrets.APPLE_ISSUER_ID }}"
+          APPLE_KEY_CONTENT: "${{ secrets.APPLE_KEY_CONTENT }}"
+          SLACK_URL: "${{ secrets.SLACK_WEBHOOK_DEVOPS }}"
+          MATCH_PASSWORD: "${{ secrets.MATCH_PASSWORD }}"
+          APPLICATION_THEME: ${{ inputs.APPLICATION_THEME }}
+          TESTFLIGHT_INTERNAL_TESTERS_GROUP: ${{ inputs.TESTFLIGHT_INTERNAL_TESTERS_GROUP }}
+          TESTFLIGHT_BETA_APP_DESCRIPTION: ${{ inputs.TESTFLIGHT_BETA_APP_DESCRIPTION }}
+
+      - uses: 8398a7/action-slack@v3
+        if: failure()
+        with:
+          status: ${{ job.status }}
+          fields: repo,message,author,commit,workflow,job
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_URL }}
+
+      - name: Upload Runner Log on Failure
+        if: failure()
+        uses: actions/upload-artifact@v3.1.1
+        with:
+          name: runner-log
+          path: /Users/runner/Library/Logs/gym/
+          retention-days: 1
```

Comment:
* Not required for ios-publishing

Reply:
We need this since sometimes, the certificates will be expired after some period and we need this to check what went wrong in the build process.